string Number
string Prime
string InputID
string OutputID

